infra/lambda/alert_to_webhook/index.py

```
@@ -0,0 +1,108 @@
+            errno = getattr(reason, "errno", None)
+            if errno in (110, 111, 113):  # Connection timed out, refused, No route to host
+                return True
+        # 잘못된 URL, host not found 등 -> 재시도 안 함
+        return False
+    # 알 수 없는 예외는 무한 재시도 방지를 위해 재시도 안 함
+    return False
+
+
+def lambda_handler(event, context):
+    try:
+        if "Records" in event and event["Records"]:
```

Ruff (0.15.6)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)